src/extension/gutters.ts

```diff
@@ -61,6 +61,7 @@ export class Gutters {
                 commands.executeCommand("livePreview.start.internalPreview.atFile", pickedReport, false);
             }
 
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
interesting, given the rule `"@typescript-eslint/no-explicit-any": "off",` I would have thought tslint would ignore these `error: any` types 🤔.

Reply:
```json
    "overrides": [
        {
            "files": "**/test/**/*.test.ts",
            "rules": {
                "@typescript-eslint/no-explicit-any": "off",
                "@typescript-eslint/no-empty-function": "off",
                "@typescript-eslint/no-unused-vars": "off"
            }
        }
    ]
```

These three rules above, are turned off only for `test` folder, while they are still applied across the project, i.e. in `src`. That's why I have to manually disable `@typescript-eslint/no-explicit-any` in `gutters.ts` and `coverageparser.ts`.

Let me explain, why I did so, it's because of most of the warnings related to these three rules were produced by `*.test.ts` files.

Please let me know, if this is not desired or we should turn off them across the project instead of only in `test` directory. I will make a proper changes.